ros2pkg/ros2pkg/verb/create.py

```diff
@@ -41,6 +41,7 @@ def add_arguments(self, parser, cli_name):
             help='The package name')
         parser.add_argument(
             '--package_format',
+            '--package-format',
```

Comment:
I suggest swapping the order here so the hyphenated one is shown first in `--help`.

Reply:
Done b8ff16d